src/colvarproxy_io.cpp

```diff
@@ -237,15 +286,16 @@ bool colvarproxy_io::input_stream_exists(std::string const &input_name)
 int colvarproxy_io::close_input_stream(std::string const &input_name)
 {
   if (colvarproxy_io::input_stream_exists(input_name)) {
-    std::ifstream *ifs = dynamic_cast<std::ifstream *>(input_streams_[input_name]);
+    std::ifstream *ifs =
+      dynamic_cast<std::ifstream *>(input_streams_[input_name]);
```

Comment:
Are these lines so long that they need to be chopped? I find it harder to read... what's the criterion?

Reply:
They would go beyond column 80. That continuation style is what `clang-format` would use if we were enforcing it.

Reply:
The problem with this type of conventions is that most people agree that the old one is bad, but disagree on a replacement.

That said, of the two MD engines that specify one (GROMACS and LAMMPS) they seem to agree on 100, so I'm happy to fall in line.